src/interfaces/IERC7540.sol

```diff
@@ -109,36 +126,39 @@ interface IERC7540Deposit {
 }
 
 interface IERC7540Redeem {
-    event RedeemRequest(address indexed sender, address indexed operator, address indexed owner, uint256 shares);
+    event RedeemRequest(address indexed sender, address indexed receiver, address indexed owner, uint256 shares);
 
     /**
      * @dev Assumes control of shares from sender into the Vault and submits a Request for asynchronous redeem/withdraw.
```

Comment:
I would suggest removing the withdraw terminology from this interface as its confusing as we only support redeem.